policies/branch-policy.md

```diff
@@ -0,0 +1,89 @@
+Branch Policy
+=============
+
+The openssl repository contains the following maintained branches:
+
+### The master branch
+
+- Any type (bug fix, feature, refactoring, ...) of pull requests is allowed.
+- The development of the next minor or major release happens there.
+  Which kind of release it is is defined by OMC at some point after the
+  master branch is opened after the previous release was branched.
+  Implicitly it means that any API/ABI breaking changes are not allowed
+  on the master branch unless OMC defines the next release will be a major
+  release.
+- Any changes merged to the master branch must be ported to the future major
+  and future minor branches if they are applicable. This can be done by
+  directly cherry-picking the changes when merging if there are no conflicts.
+  By this we must ensure that no features or bug fixes are unintentionally
+  lost in future major or minor releases.
+
+### The supported release branches
+
+- The development of the next patch releases of supported minor releases
+  happens there.
+- According to [stable release update policy] only bug fixes and
+  documentation changes are allowed.
+- By exception given by OMC also other types of pull requests can be merged.
```

Comment:
Should this exception be decided by OTC?

Reply:
IMO OTC can give an exception only if some change is necessary for fixing a bug even if that change by itself could be seen as a feature.